flink-runtime/pom.xml

```diff
@@ -117,6 +117,16 @@ under the License.
 		</dependency>
 
 		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-log4j12</artifactId>
```

Comment:
I found a solution by manually setting the dependencies to scope compile within IntelliJ. For the others who run into the same problem you can do this under: Project structure --> Modules --> Dependency tab.